src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java

```diff
@@ -162,14 +162,15 @@ private ContentMappings(@Nonnull XmlProxy proxy) {
 
     private static Set<String> getAdditionalStopWords() {
         Set<String> words = new HashSet<>();
-        String fileLocation = System.getProperty(ADDITIONAL_STOP_WORDS_PROPERTY, SupportPlugin.getRootDirectory()
-            + "/" + ADDITIONAL_STOP_WORDS_FILENAME);
-        if (fileLocation != null) {
-            LOGGER.log(Level.FINE, "Attempting to load user provided stop words from ''{0}''.", fileLocation);
-            File f = new File(fileLocation);
-            if (!f.exists() || !f.canRead()) {
-                LOGGER.log(Level.WARNING, "Could not load user provided stop words as " + fileLocation 
-                    + " does not exist or is not readable.");
+        String fileLocationFromProperty = System.getProperty(ADDITIONAL_STOP_WORDS_PROPERTY);
```

Comment:
Why did you split this into two more complex sentences? I prefer the simplest way if there is no need to change it.

Reply:
If the system property is used, but the path provided does not exist, we want to log a warning.
If the default file location does not exist, we do not want to log a warning.
In both cases those, the logic will be the same. There is just a particular scenario (system property used but file not exist where we want to log something).
Does that make sense ?

Reply:
I see now, thanks